src/plugins/autofill/__tests__/autofill.spec.js

```diff
@@ -324,27 +324,258 @@ describe('AutoFill', () => {
     expect(isFillHandleVisible()).toBe(true);
```

Comment:
Some unrelated(?) tests seem to be failing after #8067 

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/7091471/119130237-3bd9da80-ba38-11eb-8bc7-0a1be4b602df.png">

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/0790eb8f17b0bc942a7a995b9a2dfe362d425383